### PR TITLE
updating sig-cloud-provider/azure OWNERS

### PR DESCRIPTION
**Pull request by marosset**
- adding @marosset, @jackfrancis, @mboersma as reviewers
- adding @CecileRobertMichon as approver
- moving @brendandburns, @khenidak to emeritus

/assign @feiskyer @andyzhangx

### CODE REVIEW

File: config/jobs/kubernetes/sig-cloud-provider/azure/OWNERS [hunk 1]
```diff
@@ -2,19 +2,21 @@
 
 reviewers:
 - andyzhangx
-- brendandburns
 - CecileRobertMichon
 - feiskyer
+- jackfrancis
 - karataliu
```
Comment (marosset) [inline config/jobs/kubernetes/sig-cloud-provider/azure/OWNERS:8]: Does anyone know if @karataliu is still involved / should stay in the ONWERS file?

**Comment by CecileRobertMichon** [comment]
lgtm

@feiskyer anyone else that should be in the OWNERs file?

File: config/jobs/kubernetes/sig-cloud-provider/azure/OWNERS [hunk 2]
```diff
@@ -2,19 +2,21 @@
 
 reviewers:
```
Comment (CecileRobertMichon) [inline config/jobs/kubernetes/sig-cloud-provider/azure/OWNERS:3]: let's add @Jont828 and @jsturtevant as well?

**Review by andyzhangx — APPROVED** [review]
/lgtm

File: config/jobs/kubernetes/sig-cloud-provider/azure/OWNERS (hunk 2 above)
Comment (marosset) [reply config/jobs/kubernetes/sig-cloud-provider/azure/OWNERS:3]: I'll open a follow-up PR for this later today

**Pull request merged**